### PR TITLE
Bookstore: Use Redux in React components.

### PR DESCRIPTION
**Pull request by lojacho**
In this pull request, I was able to : 
- Keep states immutables in reducers.
- Change default state in books reducer.
- Import Redux Provider and store, it in the main component of React app.
- List of books displays books from the Redux store.
- Add functionality for "Add new" book.
- Add functionality for "Remove" a book.
- Add functionality for the "Check status" button

### CODE REVIEW

File: src/Components/categories.jsx [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
+  return (
+    <button
+      type="button"
+      onClick={() => dispatch(checkStatus())}
+    >
+      Check status
+    </button>
+  );
 }
-export default Categories;
```
Comment (julie-ify) [inline src/Components/categories.jsx]: - [ ] After dispatching the action to the store, you are required to fetch the "under construction" text from the store and display it on the Categories page. Kindly implement that.

  Kindly ensure that when the button is clicked the text should be displayed.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @dgcuenca,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/categories.jsx (hunk 1 above)
Comment (lojacho) [reply src/Components/categories.jsx]: Ok, thanks for the feedback, I checked the instruction for this fetch requirement but can not find anything.

**Review by Gambit142 — APPROVED** [review]
Hi @dgcuenca 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Use of _**redux**_ hooks ✔️ 
- React components are wrapped in the `Provider` component ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**